### PR TITLE
[GH-993] Fix csv export header

### PR DESCRIPTION
**Pull request by b4sen**
#### Summary
The exported CSV headers weren't consistent with the displayed ones, now they are the same.

#### Ticket Link
[#993](https://github.com/mattermost/focalboard/issues/993)

### CODE REVIEW

File: webapp/src/csvExporter.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ class CsvExporter {
 
         {
             // Header row
-            const row: string[] = ['Title']
+            const row: string[] = ['Name']
```
Comment (sbishel) [inline webapp/src/csvExporter.ts:56]: This should grab the text from the translations.

Comment (sbishel) [reply webapp/src/csvExporter.ts]: No, we squash them when we merge the PR.

**Review by sbishel — APPROVED** [review]
Awesome. Thanks @b4sen

File: webapp/src/csvExporter.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ class CsvExporter {
 
         {
             // Header row
-            const row: string[] = ['Title']
+            const row: string[] = [intl.formatMessage({id: 'TableComponent.name', defaultMessage='Name'})]
```
Comment (sbishel) [inline webapp/src/csvExporter.ts:56]:
```suggestion
            const row: string[] = [intl.formatMessage({id: 'TableComponent.name', defaultMessage: 'Name'})]
```

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @b4sen

**Pull request merged**